ui/src/components/executions/Logs.vue

```diff
@@ -169,18 +255,34 @@
             },
             onChange() {
                 this.$router.push({query: {...this.$route.query, q: this.filter, level: this.level, page: 1}});
+                if(this.raw_view) this.filterUpdateCount++
             },
             expandCollapseAll() {
                 this.$refs.logs.toggleExpandCollapseAll();
             },
             setRawView() {
                 this.raw_view = !this.raw_view;
-                if(this.raw_view) {
-                    this.$store.dispatch("execution/loadLogs", {
-                        executionId: this.execution.id,
-                        minLevel: this.level
-                    })
+                if(this.raw_view && this.temporalCursor!==undefined)      
```

Comment:
This is used when we switch from temporal to default view or vice versa  scroll to previous temporal cursor or log cursor